### PR TITLE
Separation of tempest prepare

### PR DESCRIPTION
**Pull request by monroid**

### CODE REVIEW

File: fuel_test/prepare_tempest_ci.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+            remote.check_stderr(
+                'mount %s:/var/lib/glance/images /var/lib/glance/images' % host)
+        else:
+            install_packages(remote, 'nfs-common portmap')
+            remote.check_stderr(
+                'mount %s:/var/lib/glance/images /var/lib/glance/images' % host)
+
+    def _make_shared_storage(self, nfs_server, nfs_clients, access_network):
+        self.tempest_share_glance_images(access_network)
+        switch_off_ip_tables(self.remote())
```
Comment (bogdando) [inline fuel_test/prepare_tempest_ci.py:92]: Which hosts iptables off? Shouldn't we test fuel nodes with firewall on, as it supposed to be?

File: fuel_test/settings.py [hunk 2]
```diff
@@ -113,7 +116,7 @@
 EXIST_TAR = os.environ.get('EXIST_TAR', None)
 CREATE_SNAPSHOTS = os.environ.get('CREATE_SNAPSHOTS', 'true') == 'true'
 CLEAN = os.environ.get('CLEAN', 'true') == 'true'
-ISO_IMAGE = os.environ.get('ISO_IMAGE', '~/fuel-centos-6.3-x86_64.iso')
+ISO_IMAGE = os.environ.get('ISO_IMAGE', '/home/alan/fuel-centos-6.4-x86_64-3.0.iso')
```
Comment (bogdando) [inline fuel_test/settings.py]: Who is alan?

File: deployment/puppet/openstack/manifests/mirantis_repos.pp [hunk 3]
```diff
@@ -47,31 +47,12 @@
...
-        }
-
-        apt::source { 'precise-fuel-folsom':
-          location    => 'http://download.mirantis.com/precise-fuel-folsom',
-          release     => 'precise-2.1.0.1',
-          repos       => 'main',
+          location    => 'http://download.mirantis.com/precise-fuel-grizzly',
+          release     => 'precise',
+          repos       => 'free',
           key         => 'F8AF89DD',
```
Comment (monroid) [inline deployment/puppet/openstack/manifests/mirantis_repos.pp:53]: This changes have already been discussed with Andrey

File: fuel_test/prepare_tempest_ci.py (hunk 1 above)
Comment (monroid) [inline fuel_test/prepare_tempest_ci.py:92]: This code is left to support older versions, the grizzly is not used

**Comment by aglarendil** [comment]
guy's please point this request to master

**Pull request closed without merging**